<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaginaPrincipal</class>
 <widget class="QWidget" name="PaginaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestión de Tienda</string>
  </property>
  <widget class="QPushButton" name="btnArticulos">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Artículos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClientes">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Clientes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVentas">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Ventas</string>
   </property>
  </widget>
  <widget class="QLabel" name="iconoCarrito">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="pixmap">
    <pixmap>:/iconoCarrito.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
